 # Single depth reading.

Header header           # timestamp is the time the depth was measured
                         # frame_id is the location of the depth reading

 float64 x     # Measurement of the Depth in meters
 float64 y     # Measurement of the Depth in meters
 float64 z     # Measurement of the Depth in meters
 float64 psi     # Measurement of the Depth in meters
 float64 u    # Measurement of the Depth in meters
 float64 v     # Measurement of the Depth in meters
 float64 w     # Measurement of the Depth in meters
 float64 r     # Measurement of the Depth in meters
 float64 bx     # Measurement of the Depth in meters
 float64 by     # Measurement of the Depth in meters
 float64 bz     # Measurement of the Depth in meters
 float64 bpsi     # Measurement of the Depth in meters


 float64 variance        # 0 is interpreted as variance unknown